lms/services/oauth2_token.py

```diff
@@ -4,6 +4,7 @@
 from sqlalchemy.orm.exc import NoResultFound
 
 from lms.models import OAuth2Token
+from lms.models.oauth2_token import Service
```

Comment:
The `Service` name is unfortunately rather generic here. Better suggestions?

Reply:
Yeah it's a tricky one, `scope`? Or maybe just make it more verbose


ServiceName
ServiceScope
APIName (ugly)